integrations/integrations.go

```diff
@@ -273,14 +270,25 @@ func startOPDevnet(
 		return fmt.Errorf("parse op node url: %v", err)
 	}
 
-	if l1UserAddress := v.GetString(flagL1UserAddress); l1UserAddress != "" {
-		l1Allocs.Accounts[l1UserAddress] = state.DumpAccount{
-			Balance: "0x152D02C7E14AF6800000", // 100,000 ETH
+	if l1UserAddressStr := v.GetString(flagL1UserAddress); l1UserAddressStr != "" {
+		if !common.IsHexAddress(l1UserAddressStr) {
+			return errors.New("l1 user address is not a valid hex address")
+		}
+		balance, ok := new(big.Int).SetString("0x152D02C7E14AF6800000", 0) // 100,000 ETH
+		if !ok {
+			return errors.New("failed to parse balance as big int")
+		}
```

Comment:
nit: Since we no longer need an explicit string type here for the balance it may be more readable to just create the bigInt directly with something like `big.NewInt(100_000 * ETH)`

Reply:
Unfortunately, `100_000 * ETH` overflows the int type.